tests/e2e/util.py

```diff
@@ -13,11 +12,14 @@
 operator_label = "-l app=clickhouse-operator"
 
 
-def get_full_path(test_file):
-    return os.path.join(current_dir, f"{test_file}")
+def get_full_path(test_file, baremetal=True):
+    if baremetal:
```

Comment:
look like totally wrong code part, please remove it

Reply:
In case we run in docker, paths are different

Reply:
Ok. Accept it 
when we run tests outside docker with minikube inside, then local relative config path, have different absolute path in docker, absolute path calculates locally, but kubectl and other tools run inside docker and need other absolute path